sources/packages/backend/apps/api/src/route-controllers/education-program-offering/education-program-offering.institutions.controller.ts

```diff
@@ -158,11 +159,15 @@ export class EducationProgramOfferingInstitutionsController extends BaseControll
     description:
       "Program to create the offering not found for the institution.",
   })
+  @ApiNotFoundResponse({
+    description: "Not able to find the program in the provided location.",
+  })
   @ApiUnprocessableEntityResponse({
     description: "Not able to a create an offering due to an invalid request.",
   })
-  @ApiNotFoundResponse({
-    description: "Not able to find the program in the provided location.",
+  @ApiUnprocessableEntityResponse({
```

Comment:
As far as I remember adding multiple `ApiUnprocessableEntityResponse` will not result in the expected description in Swagger. Other codes concatenate the strings when multiple are possible, as shown below.
```ts
@ApiUnprocessableEntityResponse({
    description:
      "Not possible to create an MSFAA due to an invalid application status, or " +
      "not possible to reissue an MSFAA when there is no pending disbursements for the application, or " +
      "not possible to reissue an MSFAA when the current associated MSFAA is not cancelled.",
  })
```